internal/cmd/ssh.go

```diff
@@ -43,11 +44,23 @@ func newSSHHostsCmd(cli *CLI) *cobra.Command {
 		Args:  ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			host, port := args[0], args[1]
-			if err := runSSHHosts(cli, host, port); err != nil {
+
+			err := runSSHHosts(cli, host, port)
+			switch {
+			case errors.Is(err, errNotInfraDestination) || errors.Is(err, errBeforeDestinationMatch):
```

Comment:
The second error here is category 1 from #3927. We need a better solution for those errors, but for now I've preserved the existing behaviour of hiding them.